DTML.EduBot/Dialogs/TranslateDialog.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Bot.Builder.Dialogs;
+using Microsoft.Bot.Builder.Luis.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+
+namespace DTML.EduBot.Dialogs
+{
+    public partial class HelpDialog : QnaLuisDialog<object>
+    {
+        [LuisIntent("Translate")]
```

Comment:
We can use translate intent to explicitly ask for translation of a phase. e.g. while learning talking to the bot in English, the student can ask for translation of a specific phrase without changing the system language. 

Can we implement the call to translation api here?